File: eulerground-clojure/6.clj
;largest palindrome from product of 2 3-digit numbers
(defn palin [x] 
    (=  (apply list (str x))(reverse (str x)))
)
;(prn (palin 123)) == false

;(prn (palin 121)) == true
(def x (vector))
(doseq [i (range 999 100 -1)]
	(doseq [j (range 999 100 -1)]
		(if (palin (* i j))
			(def x (conj x (* i j)))
			)
		)
	)
(prn (first x))

File: eulerground-python/6.py
#largest palindrome from prouct of 2 3-digit numbers
def palin(num):
    strnum = str(num)
    if strnum == strnum[::-1]:
        return True
    return False
flag = True
for i in range(999,100,-1):
    if flag:
        for j in range(999,100,-1):
          if flag:
            if palin(i*j):
                print i*j
                flag = False
                break

